### PR TITLE
update(Makefile): add install and uninstall targets

### PR DESCRIPTION
**Pull request by loresuso**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

> /kind documentation

> /kind failing-test

> /kind feature

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

/area build

> /area plugin-sdk

> /area tests

**What this PR does / why we need it**:
As per PR title. INSTALL_DIR will be customizable.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -63,3 +65,11 @@ $(DEPS_INCLUDEDIR)/plugin_api.h: $(DEPS_INCLUDEDIR)
 
 $(DEPS_INCLUDEDIR)/nlohmann/json.hpp: $(DEPS_INCLUDEDIR)
 	+$(CURL) -sLo $(DEPS_INCLUDEDIR)/nlohmann/json.hpp https://github.com/nlohmann/json/releases/download/v3.10.2/json.hpp
+
+.PHONY: install
+install: 
+	cp -r $(INCLUDE_DIR) $(INSTALL_DIR)
+
+.PHONY: uninstall
+uninstall:
```
Comment (FedeDP) [inline Makefile:74]: Will this completely erase /usr/include ? 😲

Comment (leogr) [reply Makefile:74]: :exploding_head:

Comment (loresuso) [reply Makefile:74]: Yes sorry about that ahah fixing

Comment (FedeDP) [reply Makefile:74]: :rofl:

File: Makefile [hunk 2]
```diff
@@ -22,6 +22,8 @@ FALCOSECURITY_LIBS_REVISION ?= 0.11.3
 FALCOSECURITY_LIBS_REPO     ?= falcosecurity/libs
 DEPS_INCLUDEDIR             := include/falcosecurity/internal/deps
 DEPS_PLUGIN_LIB_URL         := https://raw.githubusercontent.com/${FALCOSECURITY_LIBS_REPO}/${FALCOSECURITY_LIBS_REVISION}/userspace/plugin
+INCLUDE_DIR                 := include/falcosecurity
+INSTALL_DIR                 ?= /usr/include/falcosecurity
```
Comment (FedeDP) [inline Makefile:26]: what about 
```suggestion
INSTALL_DIR                 ?= /usr/$(INCLUDE_DIR)
```

**Review by FedeDP — APPROVED** [review]
/approve

**Pull request merged**